client/src/main/java/com/battleships/gamewindow/services/RandomFleetPlacement.java

```diff
@@ -117,7 +119,7 @@ public RandomFleetPlacement() {
         return boards;
     }
 
-    public List<Coordinate> getRandomCoords() {
+    public List<Coordinate[]> getRandomCoords() {
         System.out.println("Kombinacja nr " + lastCombinationProvided % 6);
```

Comment:
That last... var modulo 6 could be extracted to local var or something, because it's not clear what this does.